sql/core/src/main/scala/org/apache/spark/sql/execution/joins/SortMergeJoinExec.scala

```diff
@@ -693,15 +701,12 @@ case class SortMergeJoinExec(
            |    continue;
            |  }
            |}
-           |if (!$loaded) {
-           |  $loaded = true;
-           |  $streamedAfter
```

Comment:
For left anti, `streamedAfter` will appear twice in the code. How large is it?

Reply:
@cloud-fan - good call for code size. Actually I just figured we don't need `$loadStreamed` here for left anti, because we can skip streamed row when there's a match here.

Reply:
@cloud-fan - updated. Also avoid unnecessary code for `bufferedAfter` and LEFT SEMI join.